examples/api/lib/widgets/scroll_view/listview_select.1.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for ListView
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: _title,
+      home: ListTileSelectExample(),
+    );
+  }
+}
+
+class ListTileSelectExample extends StatefulWidget {
+  const ListTileSelectExample({Key? key}) : super(key: key);
+
+  @override
+  ListTileSelectExampleState createState() => ListTileSelectExampleState();
+}
+
+class ListTileSelectExampleState extends State<ListTileSelectExample> {
+  bool isEditMode = false;
+  final int listLength = 30;
+  late List<bool> _selected;
+  bool _selectAll = false;
+  bool _isGridMode = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _selected = List<bool>.generate(listLength, (_) => false);
+  }
+
+  @override
+  void dispose() {
+    _selected.clear();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text(
+            'ListTile select example',
+          ),
+          leading: isEditMode
+              ? IconButton(
+                  icon: const Icon(Icons.close),
+                  onPressed: () {
+                    setState(() {
+                      isEditMode = false;
+                    });
+                  },
+                )
+              : const BackButton(),
+          actions: <Widget>[
+            if (_isGridMode)
+              IconButton(
+                icon: const Icon(Icons.grid_on),
+                onPressed: () {
+                  setState(() {
+                    _isGridMode = false;
+                  });
+                },
+              )
+            else
+              IconButton(
+                icon: const Icon(Icons.list),
+                onPressed: () {
+                  setState(() {
+                    _isGridMode = true;
+                  });
+                },
+              ),
+            if (isEditMode)
+              TextButton(
+                  child: !_selectAll
+                      ? const Text('select all')
+                      : const Text('unselect all'),
```

Comment:
Does this get clipped, or does the text all fit in the app bar?

Reply:
The text fits in appBar the title has ellipses

<img width="454" alt="image" src="https://user-images.githubusercontent.com/31410839/156592326-329600e7-3198-452d-a873-1330d2834e53.png">

Reply:
The clipping of the title is not ideal, can you update to something shorter?

Reply:
Sure, Changed to "ListTile selection"

<img width="359" alt="image" src="https://user-images.githubusercontent.com/31410839/156750828-b5632e81-0e39-4b0a-a9a1-00d0e7489c37.png">